tests/dummy/app/templates/application.hbs

```diff
@@ -1,13 +1,16 @@
 <h2 id="title">Welcome to Ember Gridstack</h2>
 
+<button class="new-item-btn" onclick={{action 'changeItems'}}>+ New Item</button>
+
 {{#grid-stack classNames='grid-stack' onChange=(action (mut test) true)}}
   {{#each items as |item|}}
-    {{#grid-stack-item}}
+    {{#grid-stack-item
+       options=(hash x=0 y=item)
```

Comment:
Is this hash just passed on?

Reply:
Yeah `x` & `y` sets the `data-gs-x` and `data-gs-y` respectively.

Reply:
There is documentation for `options`
https://github.com/yahoo/ember-gridstack#options-1

We can update it if the doc is not clear.

Reply:
oh, i was looking at the usage [here](https://github.com/yahoo/ember-gridstack/blob/master/addon/components/grid-stack-item.js#L5) and didn't see it.